tests/test.rs

```diff
@@ -17,12 +17,16 @@ use std::sync::Once;
 
 static INIT: Once = Once::new();
 
+fn test_path(name: &str) -> std::string::String {
```

Comment:
+1. This is a good change cleanup change.

I don't think the `env!("CARGO_MANIFEST_DIR")` is strictly required. I leave it out of some of the sample code. The test suite will definitely run without it. I don't know what cases, historically or otherwise, required it. Maybe you could consider dropping it here and just using `format!("tests/{name}")`.